src/main/java/dk/kb/present/DSCollection.java

```diff
@@ -87,7 +87,7 @@ public class DSCollection {
      */
     private final Map<String, View> views; // keys are lowercase
 
-    private String getRecordEndpoint;
+    private RecordTypeDto recordType;
```

Comment:
Isn't this more of a `recordRequestType` instead of just `recordType`? I was confused at first. Maybe add a bit of Javadoc to say what it is used for?

Reply:
You're right!